packaging/python/build_wheels.bash

```diff
@@ -118,17 +118,17 @@ build_wheel_osx() {
     echo " - Installing build requirements"
     cp packaging/python/build_requirements.txt my_requirements.txt
 
-    if [ "$2" == "coreneuron" ]; then
-        setup_args="--enable-coreneuron"
-        clone_nmodl_and_add_requirements
-        CMAKE_DEFS="${CMAKE_DEFS},LINK_AGAINST_PYTHON=OFF"
-    fi
-
     CMAKE_DEFS="NRN_MPI_DYNAMIC=$3"
     if [ "$USE_STATIC_READLINE" == "1" ]; then
       CMAKE_DEFS="$CMAKE_DEFS,NRN_BINARY_DIST_BUILD=ON,NRN_WHEEL_STATIC_READLINE=ON"
     fi
 
+    if [ "$2" == "coreneuron" ]; then
+        setup_args="--enable-coreneuron"
+        clone_nmodl_and_add_requirements
+        CMAKE_DEFS="${CMAKE_DEFS},LINK_AGAINST_PYTHON=OFF,CORENRN_ENABLE_OPENMP=OFF"
```

Comment:
I moved this block lower as otherwise, `CMAKE_DEFS` would be unconditionally overwritten by (now) line 121